module/VuFind/src/VuFind/Controller/SearchController.php

```diff
@@ -235,12 +235,20 @@ public function newitemAction()
             return $this->forwardTo('Search', 'NewItemResults');
         }
 
-        return $this->createViewModel(
+        $view = $this->createViewModel(
             [
+                'defaultSort' => $this->newItems()->getDefaultSort(),
                 'fundList' => $this->newItems()->getFundList(),
                 'ranges' => $this->newItems()->getRanges(),
             ]
         );
+        if ($this->newItems()->includeFacets()) {
+            $view->options = $this->serviceLocator
+                ->get(\VuFind\Search\Options\PluginManager::class)
+                ->get($this->searchClassId);
+            $this->addFacetDetailsToView($view, 'NewItems');
```

Comment:
I'm not clear what code pathway leads to this line being executed, since the only way I know to view the results page is by selecting a range at the New Items start page, and then this function short-circuits at line 235.

Reply:
The idea here is that if you go to NewItems page without any parameters, you get displayed the form where you can select range, filters, etc. That page then submits back to itself with parameters set, and it is at that point that the short-circuit redirect to the results page occurs. If you go to the `/vufind/NewItem` URL without any parameters set, this form should render. Are you seeing different results?

Reply:
Thanks for explaining (and sorry for the delay).  Yup this works fine.

Reply:
Great! Fine enough that you're willing to approve the PR, or do you want me to seek more input from others?